share/revng/test/tests/lit/mlir/backend/access.mlir

```diff
@@ -36,38 +36,38 @@ module attributes {clift.module} {
     %s = clift.local !s$p
     %u = clift.local !u$p
 
-    // CHECK: var_0->x;
+    // CHECK: _var_0->x;
     clift.expr {
       %a = clift.access<indirect 0> %s : !s$p -> !int32_t
       clift.yield %a : !int32_t
     }
 
-    // CHECK: var_1->x;
+    // CHECK: _var_1->x;

```

Comment:
Does this change belong to this commit?

Reply:
Ah right, messed up rebase! Good catch!

Reply:
Ah, no. This wasn't because of the rebase!

What happened is, the commit that added these tests was created in parallel with the PR to remove `_`, but `_`-removal went in first, so this got stuck emitting (and checking for) the `_var_` prefix.

Since this commit is where the old variable emitter is dropped, the tests also need to be changed.

Do you want a separate commit dropping `_` just before this one, maybe?